vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -1076,7 +1085,7 @@ public void onClick(View v) {
         }
 
         final ImageView iconCheck = convertView.findViewById(R.id.icon_check_invite_member);
-        iconCheck.setVisibility(participant.mIsSelectedToInvite ? View.VISIBLE : View.GONE);
+        iconCheck.setVisibility(mCurrentSelectedUsers.contains(participant.mUserId) ? View.VISIBLE : View.GONE);
```

Comment:
We should add somewhere a check on participant.mUserId before using it (see participant.isValid)
Maybe we should condition the tchap code here under "if (participant.mIsValid)" test